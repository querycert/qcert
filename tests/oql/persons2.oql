/* Returns a record (struct in OQL terminology) */
select struct(marketer: c->name, clients: d)
from c in Marketers,
     d in c->clients
