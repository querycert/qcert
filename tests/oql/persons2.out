[{"clients": "123", "marketer": "Jill Does"}, {"clients": "124", "marketer": "Jill Does"}, {"clients": "125", "marketer": "Joan Doe"}, {"clients": "102", "marketer": "Joan Doe"}, {"clients": "100", "marketer": "James Do"}, {"clients": "101", "marketer": "James Do"}]